src/plots/gl3d/scene.js

```diff
@@ -708,4 +705,11 @@ proto.toImage = function(format) {
     return dataURL;
 };
 
+proto.setConvert = function() {
+    for(var i = 0; i < 3; ++i) {
+        var ax = this.fullSceneLayout[axisProperties[i]];
+        Axes.setConvert(ax, this.fullLayout);
+    }
+};
```

Comment:
To be more precise, 3d doesn't use any of the _m and friends fields that setScale computes at the moment. So setting setScale to a noop might be a minor :racehorse: boost. But, for 3d annotations, I'm planning on using / mocking those scale fields. To be continued.